lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -1559,7 +1570,7 @@ void HdVP2BasisCurves::_UpdateRepr(HdSceneDelegate* sceneDelegate, TfToken const
 HdDirtyBits HdVP2BasisCurves::GetInitialDirtyBitsMask() const
 {
     constexpr HdDirtyBits bits = HdChangeTracker::InitRepr | HdChangeTracker::DirtyExtent
-        | HdChangeTracker::DirtyInstanceIndex | HdChangeTracker::DirtyNormals
+        | HdChangeTracker::DirtyInstancer | HdChangeTracker::DirtyNormals
```

Comment:
Does this change need to be protected by `HD_API_VERSION >= 36` condition?

Reply:
Good question. I asked that myself when I reviewed internally. :)

The GPU team said that `DirtyInstanceIndex` doesn't appear to be used by this code, so removing it doesn't matter. `DirtyInstancer` is used if `HD_API_VERSION > 36` but even if we're linking against older versions of USD/Hydra, setting it in the initial bits won't matter.